recipes/chemfiles-lib/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "4e43dbe1b1d4c3a2eae923fecc3452d2b207efc1d511f6683164d51d22b0255f" %}
+
+package:
+  name: chemfiles-lib
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
```

Comment:
Please add `license_file: LICENCE.txt` to include the license file with the package.